atlas

```diff
@@ -2,6 +2,9 @@
 # getoptions generates code that fails this check, so we need to disable it
 # shellcheck disable=SC2004
 
+# Keep the _ATLAS_VERSION line simple and consistent for grep/sed usage
+_ATLAS_VERSION="0.1.1"
```

Comment:
Should we add a comment here explaining our versioning system? I'm assuming we're going with https://semver.org/.

I also worry about needing to manually update this. I'm thinking it might make sense to get this from a tag when cutting releases. Then in the codebase we could just check for the tag and if we don't have one just show "git" or something.

Reply:
I've update the workflow and added inline comments.

Please check the new README section.